meta-ivi/conf/distro/poky-ivi-systemd.conf

```diff
@@ -28,6 +28,8 @@ VIRTUAL-RUNTIME_initscripts = ""
 
 DISTRO_FEATURES_BACKFILL_CONSIDERED = "sysvinit"
 
+SANITY_REQUIRED_UTILITIES += "java"
```

Comment:
Is it possible to specify a Java version? Some legacy tool are not working fine with new JRE...

Reply:
Hello, 
I checked but sadly, the **sanity.bbclass** doesn't allow this :disappointed:

Reply:
@sebastienraillet What is the reason for moving this line from the capicxx-native.inc file to the distro definition here?

Reply:
Hi @gunnarx,
Because if you check [the code](http://git.yoctoproject.org/cgit/cgit.cgi/poky/tree/meta/classes/sanity.bbclass#n655) of sanity.bbclass, you will see that this variable is a top level variable (like TMPDIR for example).

I also made a test by removing this line on a machine without java and yocto doesn't complain. As soon as you add it inside the distro file, yocto will complain.